beps/0001-notifications-system/README.md

```diff
@@ -113,6 +113,10 @@ The notification backend stores notification using the [database service](https:
 
 The recipients is **not** a list of users, but rather a filter that describes who should receive the notification. It must be possible to evaluate this filter in a database query, so that we can efficiently fetch all notifications for a given user. The same filter will also be used by the signal backend to determine which users should receive a signal.
 
+The read date is a timestamp of marking the notifications as read by the user. If missing, the notification is still unread.
+
+The saved status is a boolean indicating whether the user marked the notification for better visibility in the future (to "pin" the message).
```

Comment:
I think it makes sense to apply the design principle of "timestamps over booleans" here too. I.e. just like the read state, the saved state is also an optional timestamp. Although tbh it's not a super strong opinion that we need to use a timestamp here since it's arguably of little value since we already have the creation/update timestamps to sort by. Although it would allow us to highlight notifications that have been update since they were saved.

My stronger opinion here is that plain booleans are generally unnecessary and you're able to convey more information or allow for more evolvability with other data types. If we feel that the timestamp really is unnecessary then I'd instead think we could make this a saved state enum or object.

Reply:
SUre, let's consistently use the Date.

Reply:
Could you update the text to reflect that?

Reply:
done